flac.md

```diff
@@ -84,11 +84,11 @@ FLAC uses four methods for modeling the input signal:
 
 FLAC currently defines two similar methods for the coding of the error signal from the prediction stage. The error signal is coded using Exponential-Golomb codes in one of two ways:
 
-1. the encoder estimates a single Exponential-Golomb parameter based on the variance of the residual and Exponential-Golomb codes the entire residual using this parameter;
+1. the encoder estimates a single exp-golomb parameter based on the variance of the residual and exp-golomb codes the entire residual using this parameter;
```

Comment:
alternatively, perhaps we should start a definition section at the beginning (such as this one https://github.com/FFmpeg/FFV1/blob/master/ffv1.md#definitions) and define `exp-golomb` more explicitly there before using the term.